mne/io/tests/test_meas_info.py

```diff
@@ -91,22 +91,29 @@ def test_make_info():
     ch_pos = [ch['loc'][:3] for ch in info['chs']]
     assert_array_equal(ch_pos, m.pos)
 
-    names = ['nasion', 'lpa', 'rpa', '1', '2', '3', '4', '5']
-    d = read_dig_montage(hsp_fname, None, elp_fname, names, unit='m',
-                         transform=False)
-    info = create_info(ch_names=m.ch_names, sfreq=1000., ch_types='eeg',
-                       montage=d)
-    idents = [p['ident'] for p in info['dig']]
-    assert FIFF.FIFFV_POINT_NASION in idents
+
+@pytest.mark.skip(reason="setting a DigMontage is no longer the same")
+def test_make_info_with_dig_montage():
+    m = read_montage('biosemi32')
+    elp_points = _read_dig_points(elp_fname)
+    dig_mont = make_dig_montage(
+        nasion=elp_points[0], lpa=elp_points[1], rpa=elp_points[2],
+        hpi=elp_points[3:],
+        hsp=read_polhemus_fastscan(hsp_fname),
+    )
+    # info = create_info(ch_names=m.ch_names, sfreq=1000., ch_types='eeg',
+    #                    montage=dig_mont)
+    # idents = [p['ident'] for p in info['dig']]
+    # assert FIFF.FIFFV_POINT_NASION in idents
 
     info = create_info(ch_names=m.ch_names, sfreq=1000., ch_types='eeg',
-                       montage=[d, m])
+                       montage=[dig_mont, m])
```

Comment:
What does it mean to pass a list to montage ? Why not passing one using +?

Reply:
This is an old stuff it only happens in `create_info` it does not make sense to deprecate it here, it should be deprecated when deprecating montage.

here I only deprecate `set_montage`